README.md

```diff
@@ -0,0 +1,58 @@
+Codelicia\Immutable
+===================
+
+Install the `ImmutableProperties` library and get to work with immutable properties.
```

Comment:
I'd replace this with a description of the library. Installation is obvious and right below anyway